StandardCyborgNetworking/StandardCyborgNetworking/Operations/ServerUserOperations.swift

```diff
@@ -6,7 +6,7 @@
 //
 
 import Foundation
-import PromiseKit
+import class PromiseKit.Promise
```

Comment:
What does this change?

Reply:
By just importing the class we want we don't have to disambiguate the `Result` type referenced elsewhere (PromiseKit provides one too, but didn't want to have that dependency).